Submodule deps/picotls contains modified content
diff --git a/deps/picotls/include/picotls.h b/deps/picotls/include/picotls.h
index 82dd675..42e56cb 100644
--- a/deps/picotls/include/picotls.h
+++ b/deps/picotls/include/picotls.h
@@ -1278,7 +1278,7 @@ inline void ptls_buffer_init(ptls_buffer_t *buf, void *smallbuf, size_t smallbuf
 inline void ptls_buffer_dispose(ptls_buffer_t *buf)
 {
     ptls_buffer__release_memory(buf);
-    *buf = (ptls_buffer_t){NULL};
+    *buf = ptls_buffer_t{NULL};
 }
 
 inline void ptls_cipher_init(ptls_cipher_context_t *ctx, const void *iv)
